File: c#/src/exchanges/bitbay.cs
using Main;
namespace Main;

partial class bitbay : zonda
{
    public override object describe()
    {
        return this.deepExtend(base.describe(), new Dictionary<string, object>() {
            { "id", "bitbay" },
            { "alias", true },
        });
    }
}

File: js/src/bitbay.js
// ---------------------------------------------------------------------------
import zonda from './zonda.js';
// ---------------------------------------------------------------------------
export default class bitbay extends zonda {
    describe() {
        return this.deepExtend(super.describe(), {
            'id': 'bitbay',
            'name': 'BitBay',
            'alias': true,
        });
    }
}
